spec/definitions.yaml

```diff
@@ -840,10 +840,10 @@ definitions:
         type: object
         properties:
           min:
-            description: Minimum number of nodes in the pool
+            description: Minimum number of nodes in the pool, must be greater than 0, less than `max`.
             type: integer
           max:
-            description: Maximum number of nodes in the pool
+            description: Maximum number of nodes in the pool, must be greater than `min`.
```

Comment:
see above

Reply:
Since the validation of min and max depends on each other, I think we could move this to the description of the `scaling` attribute. WDYT?

Reply:
Having the description in the `scaling` part makes sense to me